server/src/routes/api/v1/characterRouter.js

```diff
@@ -38,4 +49,6 @@ characterRouter.post("/", async (req, res) => {
   }
 })
 
+characterRouter.use('/:id/vote', characterVoteRouter)
```

Comment:
I don't see a `characterVoteRouter` in this PR (this should be named with `Votes`, plural since a character has many votes

Reply:
Was not present due to merge-related conflicts.  Present now.